src/AccordionSection/AccordionHeader.js

```diff
@@ -17,17 +17,13 @@ AccordionHeaderIcon.propTypes = {
 
 const AccordionHeaderNote = ({ note }) =>
   note && (
-    <Text fontSize="size4" textAlign="right" color="text.muted">
+    <Text mr="regular" fontSize="size4" textAlign="right" color="text.muted">
```

Comment:
Should this even exist anymore? The user could just pass a node for note right?  I know in some of the examples we have been putting buttons here.

Reply:
Hmm good point. The things that seem useful here are the margin-right, which we could either set as padding on the section and the textAlignment but that could potentially just be handled by the consumer.